boards/arm/tiva/tm4c1294-launchpad/src/tm4c_can.c

```diff
@@ -0,0 +1,148 @@
+/****************************************************************************
+ * boards/arm/tiva/tm4c1294-launchpad/src/tm4c_can.c
+ * Based heavily on stm32_can.c from the boards directory.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
need use apache copyright for new file

Reply:
Done in [997e60a](https://github.com/apache/nuttx/pull/8152/commits/997e60aaddec0694f4ca171eeeae0adc57b66ab3)